package.json

```diff
@@ -29,8 +29,12 @@
     "@material/fab": "^13.0.0",
     "@mui/icons-material": "^5.5.1",
     "@mui/material": "^5.5.3",
+<<<<<<< HEAD
+    "ajv-dist": "^8.11.0",
+=======
     "@reach/combobox": "^0.17.0",
     "@react-google-maps/api": "^2.12.1",
+>>>>>>> b673632cbd518914b9aec29025de3ea2a6edcc58
```

Comment:
committed a merge conflict here

Reply:
I went ahead and edited the file so it was resolved before merging to main.